scripts/common_startup.sh

```diff
@@ -113,7 +113,7 @@ if [ $SET_VENV -eq 1 -a $CREATE_VENV -eq 1 ]; then
             # Ensure Python is a supported version before creating .venv
             python ./scripts/check_python.py || exit 1
             if command -v virtualenv >/dev/null; then
-                virtualenv -p python2.7 "$GALAXY_VIRTUAL_ENV"
+                virtualenv -p $(command -v python) "$GALAXY_VIRTUAL_ENV"
```

Comment:
We don't want this yet do we? We should default to 2.7 and allow 3 behind a flag until our support is super solid IMO.

Reply:
That's already taken care of at line 114 by `python ./scripts/check_python.py`

Reply:
Awesome then - thanks for setting me straight! This all looks fantastic to me then!